File: 백준/Bronze/13496. The Merchant of Venice/The Merchant of Venice.c
#include <stdio.h>

int main()
{
    int k, n, s, d, d_i, v_i, i, j, sum = 0;

    scanf("%d", &k);

    for (i = 0; i < k; i++) {
        scanf("%d %d %d", &n, &s, &d);
        for (j = 0; j < n; j++) {
            scanf("%d %d", &d_i, &v_i);
            if (s * d >= d_i)
                sum += v_i;
        }
        printf("Data Set %d:\n%d\n\n", i + 1, sum);
        sum = 0;
    }

    return 0;
}

File: 백준/Bronze/13496. The Merchant of Venice/The Merchant of Venice.py
k = int(input())
sum = 0
for i in range(k):
    n, s, d = map(int, input().split())
    for j in range(n):
        d_i, v_i = map(int, input().split())
        if (s * d) >= d_i:
            sum += v_i
    print("Data Set ", (i + 1), ":", sep = '')
    print(sum)
    print()
    sum = 0